src/lib.rs

```diff
@@ -8,7 +8,7 @@ pub fn plugin_init(plugin: &gst::Plugin) -> Result<(), glib::BoolError> {
 }
 
 gst::plugin_define!(
-    moq,
+    gstmoq,
```

Comment:
This breaks the plugin somehow. Took an hour to debug why it couldn't find "moqsink" lul.